app/js/app/controllers/EditController.js

```diff
@@ -114,7 +120,7 @@ define(["github/github", "app/helpers", "angulartics"], function(github, helpers
 						msg[0].setSelectionRange(strLength, strLength);
 					}
 
-					$rootScope.modal.show("Enter commit message", "Enter commit message, or accept default suggestion.", msg, [{caption: "Save", value: "save"}], onShow).then(function() {
+					$rootScope.modal.show("Enter commit message", saveText, msg, [{caption: "Save", value: "save"}], onShow).then(function() {
```

Comment:
I'm not sure this works; I didn't notice the message had changed, since it wasn't red and it was where the usual message I've already read is. If you can't work out how to make it a warning red easily, then I think we shouldn't bother.